.eslintrc.js

```diff
@@ -14,6 +15,32 @@ module.exports = {
                 'no-unused-expressions': 0,
                 'no-invalid-this': 0
             }
+        },
+        {
+            files: ['*.ts', '*.tsx'],
+            rules: {
+                // eslint-plugin-header
+                'header/header': [
+                    2,
+                    'block',
+                    [
+                        {
+                            pattern: '[\n\r]+ \\* Copyright \\([cC]\\) \\d{4}(-\\d{4})? .*[\n\r]+',
+                            template: `********************************************************************************
+  * Copyright (c) ${year} EclipseSource and others.
```

Comment:
why did we remove the year range?

Reply:
The year range is still accepted by the pattern: `... \\d{4}(-\\d{4})? ...`.
The lower part is the template header that is inserted by eslint if the header is missing.
E.g. if a new typescript file is created and saved, eslint creates the header based on this template and hence uses the current year for the header.